### PR TITLE
neofs-cli: bugfix expire-at flag for session create command.

### LINKED ISSUE #2539: expire-at param for a session token doesn't have any effect
Relates to https://github.com/nspcc-dev/neofs-testcases/pull/621

## Expected Behavior
Create session token with expire_at param=current_epoch + 1 and expect it to expire after 2 epoch ticks

## Current Behavior
No expiration at all, tried different param values. Creating session token with 'lifetime' param works ok.

## Steps to Reproduce (for bugs)
Allure - ```test_session_token_expiration_flags``` 
[allure.tar.gz](https://github.com/nspcc-dev/neofs-node/files/12492120/allure.tar.gz)

## Regression
(No)

## Your Environment
Run on current latest master of neofs-node - https://github.com/nspcc-dev/neofs-node/commit/98717120a74c62b328b45f036f770b02685fb9b8

### PR DESCRIPTION
**Pull request by AliceInHunterland**
Added check of passing `--lifetime` flag by the user and handling of setting `defaultLifetime` value

Closes #2539.

### CODE REVIEW

File: cmd/neofs-cli/modules/session/create.go [hunk 1]
```diff
@@ -68,18 +74,21 @@ func createSession(cmd *cobra.Command, _ []string) {
 	endpoint, _ := cmd.Flags().GetString(commonflags.RPC)
 	currEpoch, err := internalclient.GetCurrentEpoch(ctx, endpoint)
 	common.ExitOnErr(cmd, "can't get current epoch: %w", err)
+	cmd.Println("Current epoch:", currEpoch)
```
Comment (roman-khimov) [inline cmd/neofs-cli/modules/session/create.go:77]: In verbose mode only, please. Or don't do this at all.

File: cmd/neofs-cli/modules/session/create.go [hunk 2]
```diff
@@ -68,18 +74,21 @@ func createSession(cmd *cobra.Command, _ []string) {
...
+		lifetime, _ = cmd.Flags().GetUint64(commonflags.Lifetime)
+		exp = currEpoch + lifetime
 	}
 	if exp == 0 {
 		exp = currEpoch + lifetime
 	}
 	if exp <= currEpoch {
 		common.ExitOnErr(cmd, "", errors.New("expiration epoch must be greater than current epoch"))
 	}
+	cmd.Println("Expiration epoch:", exp)
```
Comment (roman-khimov) [inline cmd/neofs-cli/modules/session/create.go:91]: Ditto.

File: cmd/neofs-cli/modules/session/create.go [hunk 3]
```diff
@@ -68,18 +74,21 @@ func createSession(cmd *cobra.Command, _ []string) {
 	endpoint, _ := cmd.Flags().GetString(commonflags.RPC)
 	currEpoch, err := internalclient.GetCurrentEpoch(ctx, endpoint)
 	common.ExitOnErr(cmd, "can't get current epoch: %w", err)
+	cmd.Println("Current epoch:", currEpoch)
 
 	exp, _ := cmd.Flags().GetUint64(commonflags.ExpireAt)
 	lifetime := uint64(defaultLifetime)
```
Comment (roman-khimov) [inline cmd/neofs-cli/modules/session/create.go:80]: Likely you don't need this var

File: cmd/neofs-cli/modules/session/create.go [hunk 4]
```diff
@@ -68,18 +74,21 @@ func createSession(cmd *cobra.Command, _ []string) {
...
-	if lfArg, _ := cmd.Flags().GetUint64(commonflags.Lifetime); lfArg != 0 {
-		exp = currEpoch + lfArg
+	if cmd.Flags().Changed(commonflags.Lifetime) {
+		lifetime, _ = cmd.Flags().GetUint64(commonflags.Lifetime)
+		exp = currEpoch + lifetime
 	}
 	if exp == 0 {
```
Comment (roman-khimov) [inline cmd/neofs-cli/modules/session/create.go:85]: And this can be an `else if`

**Review by roman-khimov — CHANGES_REQUESTED** [review]

File: cmd/neofs-cli/modules/session/create.go [hunk 5]
```diff
@@ -68,18 +74,21 @@ func createSession(cmd *cobra.Command, _ []string) {
 	endpoint, _ := cmd.Flags().GetString(commonflags.RPC)
 	currEpoch, err := internalclient.GetCurrentEpoch(ctx, endpoint)
 	common.ExitOnErr(cmd, "can't get current epoch: %w", err)
+	cmd.Println("Current epoch:", currEpoch)
 
 	exp, _ := cmd.Flags().GetUint64(commonflags.ExpireAt)
 	lifetime := uint64(defaultLifetime)
-	if lfArg, _ := cmd.Flags().GetUint64(commonflags.Lifetime); lfArg != 0 {
-		exp = currEpoch + lfArg
+	if cmd.Flags().Changed(commonflags.Lifetime) {
```
Comment (cthulhu-rider) [inline cmd/neofs-cli/modules/session/create.go:81]: IINW may be shortened to
```go
	var exp uint64
	if cmd.Flags().Changed(commonflags.ExpireAt) {
		// Cobra guarantees that commonflags.Lifetime is unset then (see init)
		exp, _ = cmd.Flags().GetUint64(commonflags.ExpireAt)
	} else {
		lifetime, _ := cmd.Flags().GetUint64(commonflags.Lifetime)
		exp = currEpoch + lifetime
	}
	if exp <= currEpoch {
		common.ExitOnErr(cmd, "", errors.New("expiration epoch must be greater than current epoch"))
	}
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -39,6 +39,7 @@ minor release, the component will be purged, so be prepared (see `Updating` sect
 - `neofs-lens write-cache list` command duplication (#2505)
 - `neofs-adm` works with contract wallet in `init` and `update-contracts` commands only (#2134)
 - Missing removed but locked objects in `SEARCH`'s results (#2526)
+- `--exire-at` flag in `neofs-cli session create` command (#2539)
```
Comment (cthulhu-rider) [inline CHANGELOG.md:42]:
```suggestion
- `--expire-at` flag in `neofs-cli session create` command (#2539)
```
btw was it broken in the latest release? if not, then this record is redundant

File: cmd/neofs-cli/modules/session/create.go (hunk 5 above)
Comment (AliceInHunterland) [reply cmd/neofs-cli/modules/session/create.go:79]: I think we also need a check of passing --lifetime 0

Comment (cthulhu-rider) [reply cmd/neofs-cli/modules/session/create.go]: suggest to add separate check for this

Comment (cthulhu-rider) [reply cmd/neofs-cli/modules/session/create.go]: technically enough, but smth like `lifetime must not be zero` would be more clearer, right?

File: cmd/neofs-cli/modules/session/create.go [hunk 7]
```diff
@@ -30,8 +31,13 @@ const defaultLifetime = 10
 var createCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create session token",
-	Args:  cobra.NoArgs,
-	Run:   createSession,
+	Long: `Create session token.
+
+Default lifetime of session token is ` + strconv.Itoa(defaultLifetime) + ` epochs 
+if one of --` + commonflags.ExpireAt + ` or --` + commonflags.Lifetime + ` flags is not specified.
```
Comment (carpawell) [inline cmd/neofs-cli/modules/session/create.go:37]:
```suggestion
if none of --` + commonflags.ExpireAt + ` or --` + commonflags.Lifetime + ` flags is specified. 
```

Double-check, please

File: cmd/neofs-cli/modules/session/create.go [hunk 8]
```diff
@@ -69,12 +75,11 @@ func createSession(cmd *cobra.Command, _ []string) {
 	currEpoch, err := internalclient.GetCurrentEpoch(ctx, endpoint)
 	common.ExitOnErr(cmd, "can't get current epoch: %w", err)
 
-	exp, _ := cmd.Flags().GetUint64(commonflags.ExpireAt)
-	lifetime := uint64(defaultLifetime)
-	if lfArg, _ := cmd.Flags().GetUint64(commonflags.Lifetime); lfArg != 0 {
-		exp = currEpoch + lfArg
-	}
-	if exp == 0 {
+	var exp uint64
```
Comment (carpawell) [inline cmd/neofs-cli/modules/session/create.go:78]: it's the first time i see `Changed` used. doesn't 
```go
var exp uint64
if exp, _ = cmd.Flags().GetUint64(commonflags.ExpireAt); exp == 0 {
    lifetime, _ := cmd.Flags().GetUint64(commonflags.Lifetime)
    exp = currEpoch + lifetime
}
```

do the job?

**Review by carpawell — COMMENTED** [review]
Commit header should not end with a period.

**Review by carpawell — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**